http_proxy_errors.go

```diff
@@ -56,10 +57,18 @@ func (hp *HTTPProxy) errorResponse(req *http.Request, err error) *http.Response
 
 	hp.metrics.error(label)
 
-	resp := proxyutil.NewResponse(code, bytes.NewBufferString(msg+"\n"), req)
-	resp.Header.Set(ErrorHeader, err.Error())
+	var body bytes.Buffer
+	body.WriteString(hp.config.Name)
+	body.WriteString(" ")
+	body.WriteString(strings.ToLower(msg))
```

Comment:
How about we fixed the messages so that they do not start with upper case?

Reply:
ok